export_results.py

```diff
@@ -244,69 +274,113 @@ def export_binary_eur(cluster_idx, num_clusters=10, batch_size = 256):
+            
     print(mt2.describe())
-    
-    batch_idx = 1
-    get_export_path = lambda batch_idx: f'{ldprune_dir}/release/{trait_category}/{"-".join(pop_list)}_batch{batch_idx}/subbatch{cluster_idx}'
-
     while hl.hadoop_is_dir(get_export_path(batch_idx)):
         batch_idx += 1
-    print(f'\nExporting {len(cluster_pheno_id_list)} phenos to: {get_export_path(batch_idx)}\n')
+    print(f'\nExporting {new_ncol} phenos to: {get_export_path(batch_idx)}\n')
     hl.experimental.export_entries_by_col(mt = mt2,
-                                          path = get_export_path(batch_idx),
-                                          bgzip = True,
-                                          batch_size = batch_size,
-                                          use_string_key_as_file_name = True,
-                                          header_json_in_file = False)
-    end = time()
-    print(f'\nExport complete for:\n{trait_types}\n{pop_list}\ntime: {round((end-start)/3600,2)} hrs')
-    
-def export_subset(num_pops=None, phenocode=None):
-    mt0 = get_final_sumstats_mt_for_export()
+                                            path = get_export_path(batch_idx),
+                                            bgzip = True,
+                                            batch_size = batch_size,
+                                            use_string_key_as_file_name = True,
+                                            header_json_in_file = False)
+    return batch_idx
+
+
+def export_subset(num_pops=None, phenocode=None, exponentiate_p=False, suffix=None):
+    mt0 = get_final_sumstats_mt_for_export(exponentiate_p=exponentiate_p)
     if phenocode != None:
-        print('\nFiltering to traits with phenocode: {phenocode}\n')
+        print(f'\nFiltering to traits with phenocode: {phenocode}\n')
         mt0 = mt0.filter_cols(mt0.phenocode==phenocode)
     if num_pops is None:
         for num_pops in range(1,7):
                 export_results(num_pops=num_pops, 
                                trait_types='all', 
                                batch_size=256, 
                                mt = mt0, 
-                               export_path_str=phenocode)
+                               export_path_str=phenocode,
+                               exponentiate_p=exponentiate_p,
+                               suffix=suffix)
     else:
         export_results(num_pops=num_pops, 
                        trait_types='all', 
                        batch_size=256, 
                        mt = mt0, 
-                       export_path_str=phenocode)
+                       export_path_str=phenocode,
+                       exponentiate_p=exponentiate_p,
+                       suffix=suffix)
                 
     
 def export_loo(batch_size=256, update=False):
     r'''
     For exporting p-values of meta-analysis of leave-one-out population sets
+    NOTE: Not updated for new hq meta analysis
```

Comment:
Argh. Should it be? @ktsuo remind me, the clumping uses the loo meta results, right? We'll either have to redo that or use the `meta_all` instead of the `meta_hq` which is a bummer but just a matter of how far back we want to go.

Reply:
yes I believe so

Reply:
Happy to update this. However, it looks like line 391: `meta_mt0 = meta_mt0.filter_cols(hl.len(meta_mt0.pheno_data.pop)==6)` results in this function exporting LOO results only for those phenotypes with GWAS computed for all 6 ancestry groups. Is this consistant with your understanding? That amounts to 845 phenotypes.

There are 0 phenotypes with all 6 ancestry groups passing all QC filters, 7 with 5 ancestries, 51 with 4, 136 with 3, and 330 with 2. So this filter would not export any of the hq meta analysis results. How should I proceed?

Reply:
I guess it's not worth exporting LOOs for anything <=2, but could be good to do the rest (>=3).